bigchaindb/commands/utils.py

```diff
@@ -95,6 +96,34 @@ def start(parser, argv, scope):
     return func(args)
 
 
+def mongodb_host(host):
```

Comment:
If the problem is the import at the top of the file, there is nothing wrong to put it inside the `mongodb_host` function.